src/components/PageHeader/PageHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+type PageHeaderProps = {
+  title: string;
+  className?: string;
+  standfirst?: string;
+  meta?: string;
+};
+
+export const PageHeader = ({
+  title,
+  className,
+  standfirst,
+  meta
+}: PageHeaderProps) => {
+  return (
+    <div className="title-banner" {...className}>
```

Comment:
Adding className like this won't work. Take a look at `src/components/Hamburger/Hamburger` for an example using the **classnames** utility

i.e.
```
const cssClasses = classnames('btn--nav-toggle', {
    [`${className}`]: className
  });

...
<div className={cssClasses}>
```